packages/idos-sdk-js/src/lib/data.ts

```diff
@@ -147,22 +145,8 @@ export class Data {
     record: Omit<T, "id">,
     synchronous?: boolean,
   ): Promise<Omit<T, "id"> & { id: string }> {
-    const name = `add_${this.singularize(
-      tableName === "user_attributes" ? "attributes" : tableName,
-    )}`;
-
```

Comment:
Pretty sure we need this. No? :x

Reply:
No. `name` variable was used to verify input arguments only. Now it is handled by `kwil-js` library.
We recommend to [use](https://github.com/idos-network/idos-sdk-js/blob/0093dc0651cea83730d5a1d5a90156462bb2daeb/packages/idos-sdk-js/README.md#L443) just `attributes` table name.